cg/cli/backup.py

```diff
@@ -239,7 +239,7 @@ def _get_samples(status_api: Store, object_type: str, identifier: str) -> list[S
     get_samples = {
         "sample": status_api.sample,
         "case": status_api.get_samples_by_case_id,
-        "flow_cell": status_api.get_samples_from_flow_cell,
+        "devices": status_api.get_samples_from_flow_cell,
```

Comment:
Not something that needs fixing now, but I find this dict a bit unclear. Is it to get around doing three ifs?

Reply:
this is an error will change.